t/exit.t

```diff
@@ -10,7 +10,7 @@ log_level('warn');
 repeat_each(120);
 #repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 5);
+plan tests => repeat_each() * (blocks() * 5) - (repeat_each() * 2);
```

Comment:
With the current plan, we can change the `repeat_each` value without having to change the plan.

With the suggested change, the plan has to be updated when changing the value of `repeat_each()` indeed.